templates/projects/notes/files/common/models/note.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function(Note) {
+  Note.greet = function(msg, cb) {
+    process.nextTick(function() {
+      msg = msg || 'Note';
+      cb(null, 'Model ' + msg);
+    });
+  };
```

Comment:
@ritch we are cleaning up example projects offered by `apic loopback`, as you requested in https://github.com/strongloop/loopback-workspace/issues/272.

I think the `Notes` project should not include any custom remote methods, as it should demonstrate CRUD/data access API. What do you think?

If you prefer to keep a custom remote method, then I think it should be definitely something else than `Note.greet`. What would be a better method?